Gemfile

```diff
@@ -39,7 +39,6 @@ gem "bootsnap", require: false
 
 gem "faraday"
 gem "unsplash"
-gem "jsonapi-serializer"
```

Comment:
I like the jsonapi gem, but agree with removing this and, possibly, reinstalling it as a later refactor